We ain't no DOS, and it breaks the build as MD5CTX is redeclared.

--- src/md5.h.ports.orig	Sat May  9 13:56:22 2015
+++ src/md5.h	Sat May  9 13:56:32 2015
@@ -41,9 +41,4 @@ They are here for compatibility with old (broken) vers
 int mdfileBroken(char *fn, unsigned char *digest);
 int mdbinfileBroken(char *fn, unsigned char *bindigest);
 
-/*
-* This is needed to make RSAREF happy on some MS-DOS compilers.
-*/
-typedef struct MD5Context MD5CTX;
-
 #endif /* !MD5_H */
